util/flags.go

```diff
@@ -148,6 +153,14 @@ func ProcessFlags(args []string) (*CliFlags, error) {
 	app.Flag("ignore-non-renewable-auth", "ignored; kept for compatibility").Default("false").Bool()
 	app.Flag("never-scrub", "ignored; kept for compatibility").Default("false").Bool()
 
+	// Metrics options
+	app.Flag("enable-prometheus-metrics", "enables prometheus metrics to be served on prometheus-metrics port").Default("true").BoolVar(&flags.EnablePrometheusMetrics)
+	app.Flag("prometheus-port", "specifies prometheus metrics port").Default("9192").IntVar(&flags.PrometheusPort)
```

Comment:
`9191` then let's override it in our Helm Chart to something very unique at Hootsuite